chirp/drivers/ftm3200d.py

```diff
@@ -124,10 +124,10 @@ def get_features(self):
 
     def _decode_label(self, mem):
         # TODO preserve the unknown \x80-x86 chars?
-        return str(mem.label).rstrip("\xFF").decode('ascii', 'replace')
+        return str(mem.label).rstrip("\xFF")
```

Comment:
Aren't you removing code that replaces non-ascii characters? The original use of decode to do this is probably not the best thing here, but I think you're just introducing a likely crash aren't you?

Reply:
I do not think you need to replace non-ascii characters in the memory label, since there is no way to put non-ascii characters in the memory label, right? The only problem if you try to encode a memory label, but I am working on this!

Reply:
Well, mem is a bitwise object and mem.label is a chars. Since you're not converting this to byte-clean mmap, I'm not exactly sure what happens here. What do you think happens when the image, which actually has non-ascii characters in the char field, is loaded and processed here? And why aren't you removing the `\xFF` cleaning as well. Surely the same argument would hold?

This is the problem with converting radios without real testing...

So if I use this PR to load our test image, it does not render the label at all in the regular editor. In the radio browser, memory 1 shows the unfiltered character string, which has the right-filled non-ascii characters in it. I think that means this is not doing the right thing. The tests pass because we never read and assert anything about the existing image, and we probably space-pad the label when we set it so we don't notice. Load the test image locally and check it out.